src/main/java/com/sda/zdjavapol107/checkers/CheckersSave.java

```diff
@@ -0,0 +1,88 @@
+package com.sda.zdjavapol107.checkers;
+
+import com.sda.zdjavapol107.checkers.dao.*;
+import com.sda.zdjavapol107.checkers.model.*;
+import org.hibernate.SessionFactory;
+
+import java.time.LocalDateTime;
+import java.util.*;
+
+public class CheckersSave {
+
+    SessionFactory sessionFactory;
+    private final int NUMBER_OF_PAWNS = 24;
+
+
+    CheckersSave(SessionFactory sessionFactory) {
+        this.sessionFactory = sessionFactory;
+    }
+
+    public void saveGame(CheckersBoard checkersBoard) {
+        Tournament tournament = convertTournamentToEntity(checkersBoard.getCheckersTournament());
+        Player playerOne = convertPlayerToEntity(checkersBoard.getPlayerOne());
+        Player playerTwo = convertPlayerToEntity(checkersBoard.getPlayerTwo());
+        playerOne.setTournament(tournament);
+        playerTwo.setTournament(tournament);
+        Game game = createGameEntity(playerOne, playerTwo, tournament);
+        Map<Integer, GamePeaces> gamePeacesMap = createGamePeacesMap(checkersBoard, game, playerOne, playerTwo);
+        Set<Move> setOfMoves = createSetOfMoves(checkersBoard, gamePeacesMap, playerOne, playerTwo, game);
+
+        PlayerDao playerDao = new PlayerDao(sessionFactory);
```

Comment:
mozesz ogolnie wszystkie dao wyrzucic do konstruktora :) 
jak chcesz wiedziec jak to zrobic dobrze w calym projekcie to poczytaj o dependency injection :)